src/harness/testcases/WINNF_FT_S_SSS_testcase.py

```diff
@@ -119,21 +131,63 @@ def test_WINNF_FT_S_SSS_12(self, config_filename):
     Checks that SAS UUT response with fatal alert message.
     """
     config = loadConfig(config_filename)
+    certificate_hash = getCertificateFingerprint(config['sasCert'])
+    self._sas_admin.InjectPeerSas({'certificateHash': certificate_hash,
+                                   'url': SAS_TH_URL })
     self.assertTlsHandshakeFailure(client_cert=config['sasCert'],
                                    client_key=config['sasKey'])
 
-  @winnforum_testcase
-  def test_WINNF_FT_S_SSS_13(self):
+  def generate_SSS_13_default_config(self, filename):
```

Comment:
This file should be included in this PR, right?

Reply:
Are you referring to the default configuration file of SSS.13 ? We will add in if so. Just want to clarify.